api/src/main/java/marquez/MarquezApp.java

```diff
@@ -149,6 +150,12 @@ public void registerResources(
     }
   }
 
+  @Override
+  protected void addDefaultCommands(Bootstrap<MarquezConfig> bootstrap) {
```

Comment:
The existing commands are kind of no-arg commands. Migration commands needs to be able to access database connection. That's why I included it here so that it can get `Application` as a constructor param. Within `initialize` commands are added although application is not created yet.